src/main/java/frc/robot/RobotContainer.java

```diff
@@ -122,6 +140,50 @@ private void initShooter2020() {
     }
   }
 
+  private void initLlamaNeck2022() {
+    if (RobotConstants.get().hasLlamaNeck2022()) {
+      llamaNeck = new LlamaNeck2022();
+      llamaNeck.setDefaultCommand(new LlamaNeck2022StopCMD(llamaNeck));
+      // operatorIndexRollerForward.whenHeld(new LlamaNeck2022ForwardCMD(llamaNeck));
+      // operatorIndexRollerBackward.whenHeld(new LlamaNeck2022BackwardCMD(llamaNeck));
+    }
+  }
+
+  private void initIndexer2022() {
+    if (RobotConstants.get().hasIndexer2022()) {
+      indexer = new Indexer2022();
+      indexer.setDefaultCommand(new Indexer2022StopCMD(indexer));
+      // operatorShooterShoot.whileHeld(new Indexer2022ForwardCMD(indexer));
+    }
+  }
+
+  private void initSpitter2022() {
+    if (RobotConstants.get().hasSpitter2022()) {
+      spitter = new Spitter2022();
+      spitter.setDefaultCommand(new Spitter2022StopCMD(spitter));
+      // operatorShooterFlywheel.whileHeld(new Spitter2022ForwardCMD(spitter));
+    }
+  }
+
+  private void initShooter2022() {
+    shooter2022 = new Shooter2022();
```

Comment:
If you take my suggestion above to store the other shooter subsystems in the Shooter2022 class, then you could also combine the three initXXX functions above into this one which inits the entire shooter, including its subsubsystems.

Reply:
That should be a good solution, but if we wanted to just control each subsystem individually on other robots, wouldn't that make that impossible?

Reply:
So it should only be one or the other?

Reply:
I think you answered this in the other question -- I wasn't seeing that the requirements were being handled by the individual subsystem commands. I think what you have is good.